ThreadGroup/Controllers/TGRouterAuthViewController.m

```diff
@@ -38,6 +40,9 @@ - (void)resetView {
 #pragma mark - IBActions
 
 - (IBAction)okButtonPressed:(UIButton *)sender {
+    self.view.userInteractionEnabled = NO;
```

Comment:
I'm going to piggy back this comment. There's a few additional things we should probably do here. 

- Disable 'OK' Button
- Disable textfield
- Dismiss keyboard

This also implies that we'll need to present the keyboard again if the request comes back as failed.